src/aws_signature.erl

```diff
@@ -253,6 +254,7 @@ sign_v4_query_params(AccessKeyID, SecretAccessKey, Region, Service, DateTime, Me
     URIEncodePath = proplists:get_value(uri_encode_path, Options, true),
     TimeToLive = proplists:get_value(ttl, Options, 86400),
     SessionToken = proplists:get_value(session_token, Options, undefined),
+    Tag = proplists:get_value(tag, Options, undefined),
```

Comment:
I would prefer plural of `tag` (so `tags`) both as the key in the `Options` as well as the variable name :-)

Reply:
Sure, I will update Tags, document and add a test case. Thank for reviewing!